python/lsst/pipe/tasks/makeCoaddTempExp.py

```diff
@@ -363,19 +363,16 @@ def run(self, calexpRefList, skyInfo, visitId=0):
         for calExpInd, calExpRef in enumerate(calexpRefList):
             self.log.info("Processing calexp %d of %d for this Warp: id=%s",
                           calExpInd+1, len(calexpRefList), calExpRef.dataId)
-            try:
-                ccdId = calExpRef.get("ccdExposureId", immediate=True)
-            except Exception:
-                ccdId = calExpInd
+            ccdId = calExpRef.get("ccdExposureId", immediate=True)
```

Comment:
`obs_comCam` doesn't have a `bypass_ccdExposureId`, but that's not likely to survive.   Your change is now requiring any new obs_package to write a `bypass_ccdExposureId` in order to be able to make a warp. I thought there MIGHT be some notebooks out there that make mock dataRefs.

The conversion to Gen 3 is going to move this to runDataRef anyway.